docs-src/ref-programs.scrbl

```diff
@@ -666,7 +666,16 @@ An @deftech{interaction expression}, written @reachin{interact.METHOD(EXPR_0, ..
 
 An @tech{interaction expression} may also be written @reachin{interact.KEY}, where @reachin{KEY} is bound in the @tech{participant interact interface} to a non-function type.
 
-An @tech{interaction expression} may only occur in a @tech{local step}.
+An @tech{interaction expression} may only occur in a @tech{local step}—with the exception of the following shorthand.
```

Comment:
I think you should remove this caveat and instead document the short-hand on `only`

Reply:
Moved these docs to the `only` section